microsoft-edge/devtools-guide-chromium/whats-new/2023/02/devtools-110.md

```diff
@@ -0,0 +1,175 @@
+In Microsoft Edge 110, the **Memory** tool is now much faster at taking heap snapshots.  Taking a heap snapshot is now around 70% to 86% faster.  For larger heap snapshots, we're seeing the greatest improvement in speed.  If you have noticed that taking heap snapshots is much faster, please leave us feedback in [[Feedback] Heap Snapshot Visualizer #128](https://github.com/MicrosoftEdge/DevTools/issues/128).<!-- todo: change to the real link text and URL -->
+
+![Taking a heap snapshot in the Memory tool](./devtools-110-images/faster-heap-snapshot.png)
+
+See also:
+* [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md#take-a-snapshot)
+
+
+<!-- ====================================================================== -->
+## Move tools between the Activity Bar and Quick View more easily in Focus Mode
+
+<!-- Subtitle: Customize both the Activity Bar and Quick View in Focus Mode to see only the tools you care about. -->
+
+Microsoft Edge 110 contains various improvements to **Focus Mode** in DevTools.  The following changes make it easier to customize the Activity Bar and Quick View, to show only the tools that you're currently interested in.
+
+
+<!-- ------------------------------ -->
+#### Right-click menu shows the current location of the other toolbar to move a tool to
+
+The right-click menu for moving an optional tool between Quick View and the Activity Bar now includes the position of the Activity Bar, such as:
+* **Move to bottom Quick View**
+* **Move to side Quick View**
+* **Move to left Activity Bar**
+* **Move to top Activity Bar**
+
+![Right-click menu showing 'Move to bottom Quick View'](./devtools-110-images/move-to-bottom-quick-view.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ------------------------------ -->
+#### Tool name remains visible when you right-click the tool icon
+
+The context menu for moving a tool between the Activity Bar and Quick View no longer obscures the tool's title in the toolbar:
+
+For example, the **Network Conditions** tool name remains visible when you right-click the tool's tab in Quick View:
+
+![The right-click menu in Quick View](./devtools-110-images/focus-mode-moving-tools-context-menu.png)
+
+As another example, the **Network** tool name remains visible when you right-click the tool's tab in the Activity Bar:
+
+![The right-click menu in the Activity Bar](./devtools-110-images/tool-name-remains-visible.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ------------------------------ -->
+#### Focus is preserved in Activity Bar or Quick View when moving a tool between them
+
+For keyboard shortcut and assistive technology users, focus is preserved in the Activity Bar or Quick View when moving the currently selected tool from one to the other or removing it.
+<!-- todo: is focus preserved within the tool, or within the toolbar? -->
```

Comment:
@zoherghadyali - not sure I understand "focus is preserved" in toolbar.  Is this regarding which tool is selected in the source toolbar rather than the destination toolbar?  

Can I depict this focus behavior in a screenshot?
Is it feasible to show this in a screenshot, is there a focus-box or highlighting of what's focused, in this scenario?

Reply:
It's hard to show this without a video since you are trying to show the focused tool before/after moving a tool but here's what I got:

Here, the Network tool has been moved to the bottom Quick View. The right-click menu is opened.
![image](https://user-images.githubusercontent.com/34316555/217680379-6f290c55-cc33-4f35-8ba7-28b339319d48.png)

If you now selected the "Remove from Quick View" option, then focus will go to the **Network conditions** tool which is the next tool in Quick View:
![image](https://user-images.githubusercontent.com/34316555/217680562-b9778712-1305-469f-a60f-d56553a5036e.png)

Reply:
updated text, added the pngs.  next, redo pngs...

Reply:
added todo cmts for redo's, so closing